### PR TITLE
03 pass the create home index spec

### PR DESCRIPTION
**Pull request by brandonrandall**

### CODE REVIEW

**Comment by brandonrandall** [comment]
@thedanielvogelsang @mdevoe12 @jimszalew @frustratedswede 
hey if i just submit a pr without tagging anybody, do you automatically receive a notification?

**Comment by jimszalew** [comment]
@brandonrandall I didn't notice one until I checked the little bell at the top of the screen

File: spec/features/home_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "#home index" do
+  it "goes to the home page" do
+    visit "/"
+    # expect(current_path).to eq(home)
```
Comment (jimszalew) [inline spec/features/home_index_spec.rb:6]: I think home_path might get this to pass. Not entirely sure though

**Review by jimszalew — COMMENTED** [review]
I'm not sure why the coverage folder keeps getting added if its in the gitignore. Everything looks good though.

**Pull request merged**